libexec/tgenv-upgrade

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+##########################################
+#
+# This script just works like a software
+# version updater. It's a shortcut to
+# $ git checkout master && git pull
+#
+##########################################
+
+[ -n "${TGENV_DEBUG}" ] && set -x
+source "${TGENV_ROOT}/libexec/helpers"
+
+TGFD=$HOME/.tgenv
+TGGB=$(git branch)
```

Comment:
for readability I strongly recommend avoiding acronyms and abbreviations.

suggest `current_branch` as alternative

https://google.github.io/styleguide/shellguide.html#s7.2-variable-names